docs/index.rst

```diff
@@ -133,6 +133,12 @@ sfc_climo_gen
 | **CLI**: ``uw sfc_climo_gen -h``
 | **API**: ``import uwtools.api.drivers.sfc_climo_gen``
 
+shave
+"""""
+
+| **CLI**: ``uw shave -h``
+| **API**: ``import uwtools.api.drivers.shave``
```

Comment:
```suggestion
| **API**: ``import uwtools.api.shave``
```
I'm fixing this on my UPP branch for the existing drivers, but we might as well fix this one before it merges.